### ISSUE TITLE
Configuring Google Analytics

### ISSUE THREAD

**Issue by vlavorini** [daattali/beautiful-jekyll#432]
Hello,

    I'm trying to configure this service, and I tried with:

- insert the ID at the line `google_analytics: "UA-xxxxxxxxx-y"`
- create the file gTag.js, and adding it to the line  `gtag: "myGtag.js"`

But none of the two methods appear to work.

Can you please point me to where I am doing wrong?

Thank you

V

**Comment by AzimsTech** [comment]
Me too. It just doesn't seem to work

**Comment by OCram85** [comment]
Did you checked if the script tag is included in the html head section?
Do you get any JS errors in console?

**Comment by AzimsTech** [comment]
I have merged #433, https://github.com/AzimsTech/AzimsTech.github.io/commit/6913dc424cedab1382e0a6e4548cde910c2be79c

@OCram85 
1. No Google Analytics script found on [my page](https://i.imgur.com/A5Ui6hT.png)
2. I didnt get any JS errors

**Comment by vlavorini** [comment]
There is still something wrong: Even with this mods, Google Analytics do not get signals (at least for me..)

Below a screenshot of the Fifefox inspector: the script is in both in the Head section, and in the footer. Please tell me if this is how it should be.



<img width="1010" alt="screen shot 2018-12-11 at 09 13 02" src="https://user-images.githubusercontent.com/8724860/49787441-fffaa100-fd26-11e8-8a8c-918f6774f70d.png">
<img width="1019" alt="screen shot 2018-12-11 at 09 28 04" src="https://user-images.githubusercontent.com/8724860/49787512-2fa9a900-fd27-11e8-94cc-fb095e3eb383.png">

Just for completeness, this is the G Analytics screenshot after clicking "send test traffic":
<img width="795" alt="screen shot 2018-12-11 at 09 31 29" src="https://user-images.githubusercontent.com/8724860/49787745-c70efc00-fd27-11e8-95f5-8834ad580cca.png">

**Comment by OCram85** [comment]
Like mentioned in the [docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/) the tracking script should be placed in the `<head>` section.

I don't see any problems in your snippets.

I'm just guessing:
- I's it possible to limit the tracked page hits by the domain name? - Maybe you need to `whitelist` your domain `vlavorini.github.io` in GA?
  - I had a similar issue with my matomo instance.

**Comment by OCram85** [comment]
@daattali: Hey Dean, I guess you introduced this feature because you are using it yourself. Did you ever noticed a similar behavior?

**Comment by daattali** [comment]
@vlavorini I just tested this from scratch. I forked this repository 5 minutes ago, went to google analytics and created a new Property, and added the new tracking ID into the config file of the new repo. [This is the only thing I had to do](https://github.com/daattalitest/daattalitest.github.io/commit/5ef9d560f24fb1c7db5b23b4159f362a3d52783f). I do see myself on the google analytics site.

![image](https://user-images.githubusercontent.com/952340/49825833-5da4e280-fd53-11e8-920f-2278e425856d.png)


Perhaps you somehow got your repo into a weird state and need to just start fresh?
